Proposed_First_FLEDGE_OT_Details.md

```diff
@@ -61,10 +61,13 @@ As the FLEDGE explainer talks about, the FOT#1 will include event-level reportin
 The FOT#1 will include event-level reporting for both winning and losing bids. Implementations of the `generateBid()` and `scoreAd()` worklets, provided by the buyers and sellers respectively in the auction, may call a `forDebuggingOnly.reportAdAuctionLoss()` API which takes a single string argument representing a URL. The text placeholders below will be replaced with the corresponding value from the auction when found in the reporting URL's query parameters (note that due to http://crbug.com/1338233, prior to Chrome version 107.0.5286.0, the replacements only took place in the path of the URL)
 
 *   “${winningBid}” - The value of the winning bid.  In component auctions, this value comes from the component auction and not the top-level auction.
+*   “${winningBidCurrency}” - The currency `winningBid` is in. This is `'???'` if it's in bidder's original currency, or is a currency tag if it's in seller's currency.
```

Comment:
If the `bidCurrency` and the `sellerCurrency` match, would it still be '???'?  The statement "This is `'???'` if it's in bidder's original currency" seems to indicate that.  Ditto for the next two changes.

Reply:
Yes. Except for reportWin special casing, the rule for these tags in reporting always is:
sellerCurrency set? It's sellerCurrency
sellerCurrency not set? It's ???.

So it doesn't actually convey information not in auction configuration, it's there mostly to make things easier to understand by having the interpretation hang out near the data.

Reply:
I'm confused, your "Yes" answer means it'll be "???", but your rule "sellerCurrency set? It's sellerCurrency" would indicate that it'd be the seller currency (which is the same as the buyer currency).

Reply:
No, I am confused? sellerCurrency does not have anything to do with buyer currency, whether the latter is configured in perBuyerCurrency or returned by generateBid with bidCurrency.

Reply:
So if a bid comes in with `USD` currency (i.e. auction config's `perBuyerCurrency` and generateBid()'s `bidCurrency` are both `USD`), and `sellerCurrency` is `USD`, then ${winningBidCurrency} is `???`?

Reply:
No, because sellerCurrency is set, ${winningBidCurrency} would be sellerCurrency. Hmm, the phrasing is probably not great here; it's basically a mode switch dependent on if whether sellerCurrency is set..

Reply:
Rewording queued